frontend/app/src/components/staking/liquity/LiquityStake.vue

```diff
@@ -1,12 +1,13 @@
 <script setup lang="ts">
-import { type AssetBalance } from '@rotki/common';
 import { type PropType } from 'vue';
+import { type LiquityStakingDetail } from '@rotki/common/lib/liquity';
 import StatCard from '@/components/display/StatCard.vue';
 
 defineProps({
-  stakings: {
-    required: true,
-    type: Array as PropType<AssetBalance[]>
+  stake: {
+    required: false,
+    type: Object as PropType<LiquityStakingDetail | null>,
+    default: null
```

Comment:
This is not something for this PR but for going forward can we slowly start doing:
```typescript
defineProps<{ stake: LiquityStakingDetail | null}>()
```

and if there are defaults we can use [`withDefaults`](https://vuejs.org/api/sfc-script-setup.html#default-props-values-when-using-type-declaration)

So it would look like this:
```typescript
withDefaults(
  defineProps<{ stake: LiquityStakingDetail | null}>(),
  { stake: null }
)